pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -644,7 +673,8 @@ func appendStatusCauseForBindingPluginsFeatureGateNotEnabled(field *k8sfield.Pat
 	return causes
 }
 
-func validateInterfaceNameFormat(field *k8sfield.Path, iface v1.Interface, idx int) (causes []metav1.StatusCause) {
+func validateInterfaceNameFormat(field *k8sfield.Path, iface v1.Interface, idx int) []metav1.StatusCause {
+	var causes []metav1.StatusCause
 	isValid := regexp.MustCompile(`^[A-Za-z0-9-_]+$`).MatchString
```

Comment:
nit: Move this to top of file as well?

Reply:
I find a bit confusing moving this out because it is just slightly different then the other ones but it is specific to this function only